webhook_server/libs/handlers/pull_request_handler.py

```
@@ -740,6 +740,17 @@ async def process_opened_or_synchronize_pull_request(self, pull_request: PullReq
         if self.github_webhook.conventional_title:
             setup_tasks.append(self.check_run_handler.set_conventional_title_queued())
 
+        # Queue custom check runs (only if current action matches configured triggers)
+        current_action = self.hook_data.get("action", "")
+        for custom_check in self.github_webhook.custom_check_runs:
+            check_triggers = custom_check.get("triggers", ["opened", "synchronize", "reopened"])
+            if current_action in check_triggers:
+                check_name = custom_check.get("name")
+                if not check_name:
+                    self.logger.warning(f"{self.log_prefix} Custom check missing required 'name' field, skipping")
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)